### PR TITLE
Run linters and formatters on the codebase

### PR DESCRIPTION
**Pull request by mscheltienne**
Again, PR independent from the 2 others.
I used the configuration in #23 to run the style checks (except `codespell`). The git diff was rather small actually!

### CODE REVIEW

File: steam_web_api/apps.py [hunk 1]
```diff
@@ -102,14 +103,14 @@ def search_games(self, term, country="US"):
         soup = BeautifulSoup(html, features="html.parser")
         links = soup.find_all("a")
         apps = []
-        for l in links:
-            if l.has_attr("data-ds-appid"):
+        for link in links:
```
Comment (deivit24) [inline steam_web_api/apps.py:106]: ahh yes definitely better now that more people are committing to the project

**Review by deivit24 — APPROVED** [review]
looks good to me. @mscheltienne did you use black or a specific formatted for this?

File: steam_web_api/apps.py (hunk 1 above)
Comment (mscheltienne) [reply steam_web_api/apps.py:106]: It's even an automatic flag by linters, from the top of my head, it's something long the line of 'ambiguous variable name'

**Comment by mscheltienne** [comment]
@deivit24 `ruff`, actually through VSCode extension directly. It formats similarly to `black` but is much much faster, and now preferred by many projects.

**Pull request merged**